app/index.js

```diff
@@ -30,6 +30,10 @@ export default class InfiniteScroll extends Component {
     this.el = this.props.height ? this._infScroll : window;
     this.el.addEventListener('scroll', this.throttledOnScrollListener);
 
+    if (typeof this.props.initialScrollY === 'number') {
+      this.el.scrollTo(0, this.props.initialScrollY);
```

Comment:
This will only work if you've supplied all the data rows as per the scrollY. Let's say you scrolled down and fetched new set of data three times. Then you navigated away and the component was unmounted. You navigate back and pass the previously saved scrollY but you only have first load of data.

This is highly dependent on the `children` nodes.

Reply:
True. I am using this with redux, so this never was an issue for me. The data is always available in the store after it has been fetched once.

Perhaps we should check the `scrollHeight` of the container before applying the `scrollY`?

Reply:
Yep, exactly! Can you make this change please?